src/components/hacs-repository-card.ts

```diff
@@ -251,15 +244,16 @@ export class HacsRepositoryCard extends LitElement {
   }
 
   private async _uninstallRepository() {
-    if (this.repository.category === "plugin" && this.hacs.status.lovelace_mode !== "yaml") {
-      const resources = await fetchResources(this.hass);
-      resources
-        .filter((resource) => resource.url === this._lovelaceUrl())
-        .forEach((resource) => {
-          deleteResource(this.hass, String(resource.id));
-        });
-    }
-    await repositoryUninstall(this.hass, this.repository.id);
+    this.dispatchEvent(
```

Comment:
Use `showConfirmationDialog` from `"../../homeassistant-frontend/src/dialogs/generic/show-dialog-box"` instead.

(I will eventually move all simple dialogs to that)

```
    if (
      !(await showConfirmationDialog(this, {
        title: this.hacs.localize("dialog.uninstall.title"),
        text: this.hacs.localize("dialog.uninstall.message").replace("{name}", this.repository.name),
        confirmText: this.hacs.localize("dialog.uninstall.title"),
      }))
    ) {
      return;
    }
```